by-language/python-sqlalchemy/insert_efficient.py

```diff
@@ -0,0 +1,138 @@
+"""
+About
+=====
+
+Example program to demonstrate multi-row inserts and batched inserts
+using SQLAlchemy's `insertmanyvalues_page_size` option.
+
+- https://docs.sqlalchemy.org/core/connections.html#controlling-the-batch-size
+- https://github.com/crate/crate-python/pull/539
+
+
+
+Setup
+=====
+::
+
+    pip install --upgrade 'crate[sqlalchemy]'
+
+
+Synopsis
+========
+::
+
+    # Run CrateDB
+    docker run --rm -it --publish=4200:4200 crate
+
+    # Run PostgreSQL
+    docker run --rm -it --publish=5432:5432 --env "POSTGRES_HOST_AUTH_METHOD=trust" postgres:15 postgres -c log_statement=all
+
+    # Use SQLite
+    time python insert_efficient.py sqlite multirow
+    time python insert_efficient.py sqlite batched
+
+    # Use PostgreSQL
+    time python insert_efficient.py postgresql multirow
+    time python insert_efficient.py postgresql batched
```

Comment:
How is this example related to other DB's? Why should I run this for other DB's?

Reply:
I wanted to have an easy stage to run comparisons side-by-side. It is now integrated into the relevant example program, and I find it convenient.